src/java_instance_wrapper.h

```diff
@@ -1,77 +1,113 @@
 #ifndef GODOT_JVM_JAVAINSTANCEWRAPPER_H
 #define GODOT_JVM_JAVAINSTANCEWRAPPER_H
 
-
 #include "jni/wrapper.h"
 
 #define JNI_INIT_STATICS_FOR_CLASS(C) \
-    C::JNIMethods C::jni_methods{};
+	C::JNIMethods C::jni_methods{};
 
 #define JNI_METHOD(var_name, name, signature) \
-    jni::JavaMethodSignature var_name{name, signature};
+	jni::JavaMethodSignature var_name{ name, signature };
 
 #define DECLARE_JNI_METHODS(...) \
-private: \
-    struct JNIMethods { \
-        __VA_ARGS__ \
-    }; \
-    static JNIMethods jni_methods;
+private:                         \
+	struct JNIMethods {          \
+		__VA_ARGS__              \
+	};                           \
+	static JNIMethods jni_methods;
 
-template<class Derived>
+template <class Derived>
 class JavaInstanceWrapper {
 protected:
-    jni::JObject wrapped;
-    jni::JObject class_loader;
+	bool is_weak;
+	jni::JObject wrapped;
+	jni::JObject class_loader;
+
+	static jni::JClass j_class;
 
-    static jni::JClass j_class;
+	JavaInstanceWrapper(const char *p_class_name, jni::JObject p_wrapped, jni::JObject &p_class_loader);
 
-    JavaInstanceWrapper(const char* p_class_name, jni::JObject p_wrapped, jni::JObject& p_class_loader);

```

Comment:
Should we align on godot formatting ?

Reply:
I suggest we have a separate PR to set one format style once for good.